src/Template/Pages/Modules/view.twig

```diff
@@ -115,8 +116,26 @@
                 )) %}
             {% endif %}
 
+            {# Append "Lock/Unlock" #}
+            {% if object.id and Perms.canLock() %}
+                {% if object.meta.locked %}
+                    {% do _view.append('module-buttons', Html.link(
+                        __('Unlock'),
+                        {'_name': 'lock:remove', 'object_type': objectType, 'id': object.id},
+                        {'class': 'button button-outlined button-outlined-hover-module-' ~ currentModule.name }
+                    )) %}
+                    {{ Form.hidden('locked', {'value': object.meta.locked})|raw }}
```

Comment:
are we using this `locked` form field anywhere?

Reply:
yes, it's used to remove "status" and "uname" from payload, on data save, when object is locked